app/views/components/question.scala.html

```diff
@@ -24,6 +24,8 @@
     <h1 class="govuk-fieldset__heading">@components.render_text(q.text)</h1>
   </legend>
 
+  @components.error_message(q.errorMsgs)
```

Comment:
I think this may be in the wrong place. I think the individual error messages should appear just above the  Html element associated with the errors. Therefore, the error messages should appear between the textual UI components and the radio buttons. See https://design-system.service.gov.uk/components/error-message/.

Reply:
Yes will look at this (re. hangouts discussion)